client/src/components/NewCardForm/ColorBadgePalette.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, useState } from 'react';
+import useStyles from './useStyles';
+import ColorBadge from './ColorBadge';
+import { Box } from '@material-ui/core';
+import { ColorTags } from '../../interface/Column';
+
+interface BadgePaletteProps {
+  dohandleSetColor: (color: string) => void;
+}
+
+const BadgePalette: FC<BadgePaletteProps> = ({ dohandleSetColor }): JSX.Element => {
+  const [checkedBadge, setCheckedBadge] = useState<ColorTags>('');
+  const classes = useStyles();
+  const onSelectBadge = (color: ColorTags) => {
+    dohandleSetColor(color);
+    setCheckedBadge(color);
+  };
+  return (
+    <Box className={classes.badgesContainer}>
+      <ColorBadge color="#5ACD76" checked={checkedBadge} onClick={() => onSelectBadge('green')} />
+      <ColorBadge color="#FF5D48" checked={checkedBadge} onClick={() => onSelectBadge('red')} />
+      <ColorBadge color="#EDAB1D" checked={checkedBadge} onClick={() => onSelectBadge('yellow')} />
+      <ColorBadge color="#59B0FF" checked={checkedBadge} onClick={() => onSelectBadge('blue')} />
```

Comment:
Could we refactor here to make it utilize a data structure with all our tag colors, perhaps in the theme file?

Reply:
Just did that, I made use of module augmentation to allow changes to be made to the `theme` object just like you said in slack